backend/composer/models.py

```diff
@@ -566,6 +566,8 @@ class ConnectivityStatement(models.Model):
     )
     created_date = models.DateTimeField(auto_now_add=True, db_index=True)
     modified_date = models.DateTimeField(auto_now=True, db_index=True)
+    journey_description = models.JSONField(null=True, blank=True)
```

Comment:
Suggestion: My preference would be to have just 1 attribute since they need to always be in sync.